### PR TITLE
Add release 0.140 notes

### PR DESCRIPTION
**Pull request by dain**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.140.rst [hunk 1]
```diff
@@ -12,11 +12,14 @@ General Changes
 * Fix race which can cause queries to fail with a ``REMOTE_TASK_ERROR``.
 * Optimize :func:`array_distinct` for ``array<bigint>``.
 * Optimize ``>`` operator for :ref:`array_type`.
+* Fix projection push down to not duplicate non-deterministic functions
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.140.rst]: These are not very meaningful to users. I'd rephrase as:
* Fix an optimization issue that could result in non-deterministic functions being evaluated more than once and produce unexpected results.

File: presto-docs/src/main/sphinx/release/release-0.140.rst [hunk 2]
```diff
@@ -12,11 +12,14 @@ General Changes
...
+* Fix predicate move around to not duplicate columns
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.140.rst]: What's the user-visible effect of this? Explain it in those terms.

Comment (dain) [inline presto-docs/src/main/sphinx/release/release-0.140.rst]: No.   The issue is you get duped columns into exchanges, and hash builds.  IIRC you also get extra hash computations.

Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.140.rst]: Ah, so this is the performance issue... Then I'd rephrase as:
* Improve performance of joins, aggregations, etc. by removing unnecessarily duplicated columns.
